tests/test_20_use_case_topology.py

```diff
@@ -229,6 +229,161 @@ def test_011_intra_domain_link_down_path_found(self):
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.3.1.6')
+        
+        Ampath1 = self.net.net.get('Ampath1')
+        Ampath1.intf('Ampath1-eth40').ifconfig('down') 
+
+        time.sleep(15)
+
+        data = requests.get(l2vpn_api_url).json()
+        assert data[l2vpn_id]["status"] == "up"
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.3.1.6')
+
+        ### Reset 
+        Ampath1.intf('Ampath1-eth40').ifconfig('up') 
+
+        time.sleep(15)
+
+        data = requests.get(l2vpn_api_url).json()
+        assert data[l2vpn_id]["status"] == "up"
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.3.1.6')
+
+    @pytest.mark.xfail(reason="The L2VPN status remains up after changing the status of an associated node to down")
+    def test_021_port_in_inter_domain_link_down_no_reprov(self):
+        """ 
+        Use case 2: OXPO sends a topology update with a Port Down and that port is part of an inter-domain link.
+        """
+
+        l2vpn_api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        l2vpn_payload = {
+            "name": "Test L2VPN for port down",
+            "endpoints": [
+                {
+                    "port_id": "urn:sdx:port:ampath.net:Ampath1:50",
+                    "vlan": "400",
+                },
+                {
+                    "port_id": "urn:sdx:port:tenet.ac.za:Tenet01:50",
+                    "vlan": "400",
+                }
+            ]
+        }
+        response = requests.post(l2vpn_api_url, json=l2vpn_payload)
+        assert response.status_code == 201, response.text
+        l2vpn_id = response.json().get("service_id")
+
+        # Wait for L2VPN to be provisioned
+        time.sleep(5)
+
+        response = requests.get(l2vpn_api_url)
+        assert response.status_code == 200, response.text
+        l2vpn_status = response.json().get(l2vpn_id).get("status")
+        assert l2vpn_status == "up", f"L2VPN status should be up, but is {l2vpn_status}"
+
+        h1, h6 = self.net.net.get('h1', 'h6')
+        h1.cmd('ip link add link %s name vlan400 type vlan id 400' % (h1.intfNames()[0]))
+        h1.cmd('ip link set up vlan400')
+        h1.cmd('ip addr add 10.4.1.1/24 dev vlan400')
+        h6.cmd('ip link add link %s name vlan400 type vlan id 400' % (h6.intfNames()[0]))
+        h6.cmd('ip link set up vlan400')
+        h6.cmd('ip addr add 10.4.1.6/24 dev vlan400')
+
+        # test connectivity
+        assert ', 0% packet loss,' in h1.cmd('ping -c4 10.4.1.6')
+
+        # Ampath1-eth40      
+        Ampath1 = self.net.net.get('Ampath1')
+        Ampath1.intf('Ampath1-eth40').ifconfig('down') 
+
+        #  Cause no further (re)provisioning to be possible       
+        Tenet01 = self.net.net.get('Tenet01')
+        Tenet01.intf('Tenet01-eth41').ifconfig('down') 
+        Tenet01.intf('Tenet01-eth1').ifconfig('down') 
```

Comment:
One of the ports above is a intra domain port, so in order to be more strict to the description of the use case, you should only change inter-domain ports in this test. I suggest you change those three ports above with just two:
- Tenet01 eth41
- Tenet02 eth41